.gitattributes

```diff
@@ -0,0 +1,9 @@
+# git log -L:<method>:<file>
+*.rb diff=ruby
+*.gemspec diff=ruby
+bin/* diff=ruby
+bundler/bin/* diff=ruby
+bundler/exe/* diff=ruby
```

Comment:
All the above seems completely unrelated to this changeset? If it's indeed unrelated, please drop it and explain why you need this on a separate PR.

Or at least split it into a different commit, and explain in the commit message why you need this. I've never seen these configuration on any ruby gem.

Reply:
https://github.com/ruby/ruby/blob/master/.gitattributes

https://github.com/rails/rails/blob/master/.gitattributes

Reply:
That's nice, I didn't know about it. I'm putting 

```
*.rb diff=ruby
*.gemspec diff=ruby
```

in my global `~/.git/config/attributes` so I can get the candy without having to add PRs to every project :)

Anyways, I'm good with this :+1:. Will you split this to a separate commit or PR?

Reply:
> Will you split this to a separate commit or PR?

Given that this has no affect on the anything but local use (git & RuboCop), I really don't see a need to run CI on it again.  If you really want, I can split it into two commits...  It's six lines excluding comments...

Reply:
It seems contradictory to me that you want to add changes so that we get a better git history, but refuse to split your commits so that we get a better git history. Either you care about git history, or you don't. So please, either remove the 6 lines, or split them to a separate commit.

Reply:
Done - two commits

Reply:
@MSP-Greg Thanks for splitting the commits. Part of the motivation of splitting unrelated changes into separate commits is to explain the reasons for the changes in the commit messages, so that a future reader can understand the _why_. The commit itself already contains the file being changed, and the changes being made, so please in the future add meaningful commit messages. Thanks!